pigar/cmd.py

```diff
@@ -84,13 +84,21 @@ def parse_args(args=None):
         default=['=='],
         help='The comparison operator for versions, alternatives: [==, ~=, >=]'
     )
+    parser.add_argument(
+        '--without-referenced-comments',
+        dest='minimal',
+        action='store_true',
+        help='omit requirements.txt comments showing the file and line of '
```

Comment:
The other help messages all start with an upper case, we need to keep it consistent here as well.

Reply:
@damnever Good point. I will change that right away